tests/go/clients_test.go

```diff
@@ -504,7 +504,6 @@ var _ = Describe("Client", func() {
 	})
 
 	It("Accepts `204 No Content` with empty response body", func() {
-		// Prepare the server:
```

Comment:
Can we not remove this comment? So it is align with the rest of the tests. Also add the same context as the others.

Reply:
Done.